### PR TITLE
Branch 3 - Initial landing page UI

### PR DESCRIPTION
**Pull request by vinita254**
Updated landing page content and styling. 
Once group confirms that page (layout and or content) looks fine, will add background picture possibly a little before week 7 or during week 7.

### CODE REVIEW

File: src/client/src/views/Landing Page Background.jpg
Comment (keyman015) [inline src/client/src/views/Landing Page Background.jpg:1]: Is this used in the source code at all? If not, could it please be removed to reduce clutter.

File: src/client/src/views/UWA Logo.png
Comment (keyman015) [inline src/client/src/views/UWA Logo.png:1]: Could all these images added be moved to `../public/imgs/` (probably even `../public/icons/` for better sorting of icons) to keep them in one location and not with the main pages.

File: src/client/src/views/HomeView.vue [hunk 1]
```diff
@@ -1,9 +1,194 @@
...
+        <img src="./UWA Logo.png" class="logo1" alt="University of Western Australia Logo" />
+      </a>
+      <a href="https://nlp-tlp.org/" class="logoURL" target="_blank">
+        <img
+          src="./nlp-tlp-logo.png"
+          class="logo2"
+          alt="University of Western Australia Natural & Technical Language Processing Group Logo"
+        />
+      </a>
+    </section>
```
Comment (keyman015) [inline src/client/src/views/HomeView.vue:57]: Could these logos be properly mirrored as they look uneven currently. (minor issue)

**Review by keyman015 — CHANGES_REQUESTED** [review]
Looking good so far, just had some comments about the images in general.

File: src/client/src/views/Landing Page Background.jpg
Comment (keyman015) [reply src/client/src/views/Landing Page Background.jpg:1]: Sorry, just saw your message about adding it later. Ignore this.

File: src/client/src/views/UWA Logo.png
Comment (vinita254) [reply src/client/src/views/UWA Logo.png:1]: Sure - images have now been moved to ../public/images...

File: src/client/src/views/HomeView.vue (hunk 1 above)
Comment (vinita254) [reply src/client/src/views/HomeView.vue:52]: Done. Please let me know if they still look uneven. Thanks.

File: src/client/src/views/Landing Page Background.jpg
Comment (keyman015) [reply src/client/src/views/Landing Page Background.jpg:1]: The background image needs to be removed if its not used. (currently isnt referenced/used anywhere)

**Review by keyman015 — APPROVED** [review]
Looks good. Minor issue about logo placing but it can be overlooked for later. There was mention of the background image not working properly, that can be fixed in later issue.

**Review by keyman015 — APPROVED** [review]

**Review by keyman015 — APPROVED** [review]
LGTM

**Pull request merged**